### DISCUSSION TITLE
What's the difference between this plugin and the strapi native slug field?

### DISCUSSION THREAD

**Discussion by ErhanAbi** [ComfortablyCoding/strapi-plugin-slugify#67] [Q&A] [answered]
In strapi, a slug can be obtained by defining it on model schema like this

```javascript
...
"title": {
    "type": "string",
    "required": true
},
"slug": {
    "type": "uid",
    "targetField": "title"
}
```
This will generate a slug field on the model and in the admin interface we can automatically generate slug based on title

<img width="857" alt="image" src="https://user-images.githubusercontent.com/2982657/178095694-4557c0a2-7ab7-4ea8-81f0-ef4769be53f4.png">

**Comment by ComfortablyCoding** [comment, marked as answer]
The main difference at the moment is compound slugs. You can combine multiple fields to create your slugs. The built in only works with one reference.

**Reply by ErhanAbi** [reply]
Oh, I see. Now it makes sense 🙂

**Comment by enlivn** [comment]
I find this plugin invaluable because the native slug field seems to be broken for many (including myself): https://github.com/strapi/strapi/issues/11665
